packages/fxa-content-server/app/images/default-profile.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="57.6" height="57.6" viewBox="0 0 57.6 57.6"><defs><linearGradient id="linear-gradient" x1="-3.82" y1="63.66" x2="56.35" y2="-0.64" gradientUnits="userSpaceOnUse"><stop offset="0.26" stop-color="#9059ff"/><stop offset="0.96" stop-color="#00b3f4"/></linearGradient></defs><title>default-profile</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><circle cx="28.8" cy="28.8" r="24.54" style="fill:#fbfbfe"/><path d="M28.8,0A28.8,28.8,0,1,0,57.6,28.8,28.79,28.79,0,0,0,28.8,0Zm0,8.23A12.34,12.34,0,1,1,16.46,20.57,12.33,12.33,0,0,1,28.8,8.23ZM45.73,40.34a20.39,20.39,0,0,1-33.86,0,4.15,4.15,0,0,1,.28-5.05C16.5,30.27,22.53,37,28.8,37s12.3-6.76,16.65-1.74A4.15,4.15,0,0,1,45.73,40.34Z" style="fill:url(#linear-gradient)"/></g></g></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="57.6" height="57.6" viewBox="0 0 57.6 57.6"><defs><linearGradient id="linear-gradient" x1="-3.82" y1="63.66" x2="56.35" y2="-0.64" gradientUnits="userSpaceOnUse"><stop offset="0.26" stop-color="#9059ff"/><stop offset="0.96" stop-color="#00b3f4"/></linearGradient></defs><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><circle cx="28.8" cy="28.8" r="24.54" style="fill:#fbfbfe"/><path d="M28.8,0A28.8,28.8,0,1,0,57.6,28.8,28.79,28.79,0,0,0,28.8,0Zm0,8.23A12.34,12.34,0,1,1,16.46,20.57,12.33,12.33,0,0,1,28.8,8.23ZM45.73,40.34a20.39,20.39,0,0,1-33.86,0,4.15,4.15,0,0,1,.28-5.05C16.5,30.27,22.53,37,28.8,37s12.3-6.76,16.65-1.74A4.15,4.15,0,0,1,45.73,40.34Z" style="fill:url(#linear-gradient)"/></g></g></svg>
```

Comment:
This might could be a separate commit but all I did was remove the `<title>` tag here, it wasn't hurting anything but I saw it in the DOM and it's not doing anything so I removed it. I like using `aria-label` instead since it's external to the SVG (also not sure if all screenreaders hit `title`).

Reply:
Noted, cool with me. Now that I'm looking at it, I would also recommend we consider running  SVGs through a scrubber. There's no need to have all the `id="Layer_1-2"` crud, and I think it could help with anomalies in the output (e.g. `style="fill:#fbfbfe"` could just be `fill="#fbfbfb"` I think?). I personally use [svgcleaner](https://github.com/RazrFalcon/svgcleaner-gui).

Reply:
Actually I'll run this through SVGO, I'm not sure if it was before.

The problem with automatically running SVGs through a scrubber is that you can't attach image metadata that persists in git, or at least you couldn't a couple of years ago when I tried to do just this, so we wouldn't know what's been ran through and what hasn't without some flag, and then very occasionally, throwing an SVG through one automatically can remove something it shouldn't have and it looks funky and you won't know until after it's merged and noticed in staging. 😓 I'm all ears for a solution on both fronts though because I'd love to automate that.

Reply:
Oh that makes perfect sense. Yeah. Without having looked into it, I would suspect there should be a way, a la linting ignore statements, to ignore the next line / a specific tag? If not, it sounds like a great PR for one of the tools :D

Reply:
> Without having looked into it, I would suspect there should be a way, a la linting ignore statements, to ignore the next line / a specific tag?

Maybe. We'd have to actually open/read the file and parse through the text for a particular flag (maybe a specific class name or something) and then add that if it successfully ran through, and skip any SVGs with that flag. The other problem isn't anything to scoff at though, I have been burned _many_ times by sending something through SVGO and pushing it up without checking it after processing and for some reason the problems always occured in not just IE but FF as well. If we actually build some type of CLI tool that asked you to confirm the SVG looked correct before saving it I think that could work and would be really neat, but that's a lot of work for as few SVGs as we have.